content/applications/finance/accounting/customer_invoices/cash_discounts.rst

```diff
@@ -112,68 +143,36 @@ Never
    - The **type of cash discount tax reduction** may be correctly pre-configured, depending on your
      :ref:`fiscal localization package <fiscal_localizations/packages>`.
 
-.. _cash-discounts/gain-loss-accounts:
-
-Cash discount gain/loss accounts
---------------------------------
-
-With a cash discount, the amount you earn depends on whether the customer benefits from the cash
-discount or not. This inevitably leads to gains and losses, which are recorded on default accounts.
-
-To modify these accounts, go to :menuselection:`Accounting --> Configuration --> Settings`, and in
-the :guilabel:`Default Accounts` section, select the accounts you want to use for the
-:guilabel:`Cash Discount Gain account` and :guilabel:`Cash Discount Loss account`.
-
-.. _cash-discounts/payment-terms:
-
-Payment terms
--------------
-
-Cash discounts are defined on :doc:`payment terms <payment_terms>`. Configure them to your liking by
-going to :menuselection:`Accounting --> Configuration --> Payment Terms`, and make sure to fill out
-the fields :guilabel:`Discount %` and :guilabel:`Discount Days`.
-
-.. image:: cash_discounts/payment-terms.png
-   :align: center
-   :alt: Configuration of payment terms named "2/7 Net 30". The field "Description on Invoices"
-         reads: "Payment terms: 30 Days, 2% Early Payment Discount under 7 days".
-
-.. seealso::
-   :doc:`payment_terms`
-
 .. _cash-discounts/customer-invoice:
 
 Apply a cash discount to a customer invoice
 ===========================================
 
-Apply a cash discount to a customer invoice by selecting the :ref:`payment terms you created
+On a customer invoice, apply a cash discount by selecting the :ref:`payment terms you created
 <cash-discounts/payment-terms>`. Odoo automatically computes the correct amounts, tax amounts, due
 dates, and accounting records.
 
 Under the :guilabel:`Journal Items` tab, you can display the discount details by clicking on the
 "toggle" button and adding the :guilabel:`Discount Date` and :guilabel:`Discount Amount` columns.
```

Comment:
I agree - this is another one that came from Jon so I left it, but I think it could be better. I prefer the term "kebab menu" for the name of this UI. As far as I can find, it's the most official name for that type of menu. Here's a source: https://careerfoundry.com/en/blog/ui-design/ui-element-glossary/
I understand your problem with this is that we haven't used this term in our documentation before, but I think we should start to use it. Since it's an actual name, if anyone is confused and wants to google "what is a kebab menu" - they will find it easily. But let's ask xpl or jcs whenever they do the final review :)